tests/test_protocols/test_generator.py

```diff
@@ -335,9 +336,13 @@ def test_generated_protocol_end_to_end(self):
         )
         encoded_message_2_in_bytes = TProtocolSerializer().encode(message_2)
 
-        # add handlers to AEA resources
+        # add handlers to AEA resources]
+        skill_context_1 = SkillContext(aea_1.context)
```

Comment:
this is hacky, ideally the handler is also passed to the skill, but since it is a test only i couldn't care to fix it